README.md

```diff
@@ -187,7 +187,7 @@ You will probably need to modify `host`, `data_iface`, and `io_disks` values.
   ```sh
   /opt/seagate/cortx/hare/libexec/m0crate-io-conf >/tmp/m0crate-io.yaml
   dd if=/dev/urandom of=/tmp/128M bs=1M count=128
-  sudo m0crate -S /tmp/m0crate-io.yaml
+  sudo ./motr/motr/m0crate/m0crate -S /tmp/m0crate-io.yaml
```

Comment:
This will work only if Motr is built from sources, which is not the case when people want to install binary Motr rpm package instead of building Motr themselves.

I suggest to add a comment instead of change the example script. Where you can describe that in case of Motr built from sources `m0crate` is not available in the default system `PATH` and needs to be called explicitly using full path in Motr source dir.